packages/core/src/workflows/workflow.ts

```diff
@@ -688,14 +705,6 @@ export class Workflow<
     return null;
   }
 
-  watch(onTransition: (state: WorkflowRunState) => void): () => void {
```

Comment:
We should probably keep this since people are using it already. The new API doesn't mean the old API should go away at least not yet.

Thoughts @wardpeet ?

Reply:
it's still in here, just moved further down, and with a `logger.warn` message added in